src/shared/components/virtualized-log-viewer/useTokenizationWorker.ts

```
@@ -0,0 +1,193 @@
+          const worker = new Worker(
+            new URL('./tokenization.worker.ts', import.meta.url),
+          );
+
+          worker.onmessage = this.handleWorkerMessage.bind(this);
+          worker.onerror = this.handleWorkerError.bind(this);
+
+          this.workers.push(worker);
+        }
+      } catch (error) {
+        // Worker creation failed, disable worker support
+        console.warn('Failed to create tokenization workers:', error);
```

ESLint

[error] Unexpected console statement.

(no-console)